lib/solidus_bolt/configuration.rb

```diff
@@ -2,7 +2,7 @@
 
 module SolidusBolt
   class Configuration
-    attr_accessor :bolt_api_key, :bolt_signing_secret
+    attr_accessor :bolt_api_key, :bolt_signing_secret, :bolt_publishable_key, :bolt_environment
```

Comment:
doesn't need to be part of this PR

Reply:
Similar change to the above comment
We can remove it while rebasing but rspec tests won't run without this change